readme.md

```diff
@@ -1,66 +1,66 @@
+In folder [/wp-content/themes/vost/dev](https://github.com/vostpt/current-website/tree/master/wp-content/themes/vost/dev) there will be the production environment to minification, transpile and compilation of css and js code;
 
-dentro do */src* estarão os ficheiros .php, copiados diretamente
+Inside */dev* folder make `npm install`, and then run gulp with `gulp serve`;
 
-*/src/media* são optimizadas as imagens ao iniciar o gulp serve e depois são apenas copiadas (questão de processamento de cpu - podem editar o gulpfile caso queiram sempre).
+.php files are located in */src* folder, directly copied;
 
-*/src/js/* concatenados, transpilados e minificados.
-*/src/js/* vendor copiado diretamente sem alteração
-*/src/js/* unique minificado, transpilado
+*/src/media* the images are optimized while gulp is starting, after that they're only copied (CPU processing issues - you can edit gulpfile if you want);
 
-*/src/css/unique* copiado diretamente sem merge
-*/src/css* minificado 
+*/src/js/* concatenated, transpiled and minified;
+*/src/js/* vendor copied directly without changes;
+*/src/js/* unique minified and transpiled;
 
-dentro do */src* estará a base de dados .sql depois de importada é preciso alterar no wp_options os 2 campos do endereço do site.
+*/src/css/unique* copied directly without merge;
+*/src/css* minified;
 
-Actualizar os permalinks.
+Inside */src* folder there will be .sql database. After import it's necessary to change in wp_options the 2 site url fields;
 
-## Pequenas notas
+Update permalinks.
 
-- O css está com cache buster (no header.php).
-- Se por algum motivo se desactivar o wpml, é preciso ter em atenção todas as instancias em que é chamado.
-- O mesmo para o acf mas isso ia-se logo notar :)
-- jquery só é preciso para o slickjs na home, sugestão de optimização de só fazer o call do script ai (está a chamar no functions).
-- o acf poderia-se por a carregar por json, seria ligeiramente mais eficiente
-- edição do menu em apresentação > menus
-- edicao das colunas do footer em apresentação > widgets
-- Duplicar os posts para EN (e informação)
-- no functions.php tem um snippet a remover funções para editores
-- ao fazer upload não esquecer de remover o ambiente de desenvolvimento
-- e editar o wp-config com os dados correctos de ligação à BD.
+## Small notes
 
+- Css files have cache buster (no header.php);
+- If, for some reason wpml is disabled, you must pay attention all called instances;
+- The same for acf, but you would have noticed :) ;
+- jquery is only needed for slickjs in home (optimization tip - call script only in home, it's already called in functions);
+- In order to improve efficiency, acf can be loaded using json, it would be more efficient;
+- Menu edition in apresentação > menus;
+- Footer columns edition in apresentação > widgets;
+- Duplicate posts to English (and info);
+- In functions.php exists a snippet that removes functions for editors;
```

Comment:
- There's a code snippet in the `functions.php` file that disables editor functionality;

**(not sure what the original sentence meant, I assume the above)**